geopandas/io/file.py

```diff
@@ -654,11 +653,19 @@ def _to_file(
 
 
 def _to_file_fiona(df, filename, driver, schema, crs, mode, **kwargs):
+    if not HAS_PYPROJ and crs:
+        raise ImportError(
+            "The 'pyproj' package is required to write a file with a CRS, but it is not"
+            " installed or does not import correctly."
```

Comment:
Is it ever possible to _have_ a crs but not have pyproj installed?

Reply:
Yes, you can pass it directly as a keyword argument to `GeoDataFrame.to_file`.

Reply:
Ah, I see. 
Not for this PR, but _in theory_ that could actually be a way to specify a CRS to write to the file without requiring pyproj, if the user passes the value that we pass to fiona directly (not that this is an important use cases, though, I assume)

Reply:
Yeah, you could just pass the WKT string. But the main reason to enable pyproj-less environment is to work with non-geospatial data.